pyproject.toml

```diff
@@ -93,3 +93,31 @@ convention = "numpy"
 [tool.pytest.ini_options]
 # Silence cotengra warning
 filterwarnings = ["ignore:Couldn't import `kahypar`"]
+
+[tool.mypy]
+disable_error_code = [
+  "type-arg",
```

Comment:
We can enable `type-arg` with only few changes: https://github.com/thierry-martinez/graphix/commit/d5c4d50d653082294b0eb93784cb8da46ec7145b

Reply:
I know it's not necessary for now, but would be necessary as it's really hard to resolve all the placeholders used in `npt.NDArray`: this is why I added it.
How do you think about it?

Reply:
Anyway let me adopt your idea here.